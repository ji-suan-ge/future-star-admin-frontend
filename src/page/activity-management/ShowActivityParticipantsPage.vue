<template>
  <el-container>
    <el-main>
      <el-page-header @back="goBack()" content="已参与校友名单"/>
      <!--      <el-divider></el-divider>-->
      <!--      <el-row>-->
      <!--        <el-col :span="4" :offset="3">-->
      <!--          <el-select class="selection" v-model="semester" @change="findAllClass()" filterable placeholder="请选择学期">-->
      <!--            <el-option-->
      <!--              v-for="semesters in semesterOptions"-->
      <!--              :key="semesters.id"-->
      <!--              :label="semesters.label"-->
      <!--              :value="semesters.semester">-->
      <!--            </el-option>-->
      <!--          </el-select>-->
      <!--        </el-col>-->
      <!--        <el-col :span="4">-->
      <!--          <el-select class="selection" v-model="clazz" :disabled="class_state" @change="search()" filterable placeholder="请选择班级">-->
      <!--            <el-option-->
      <!--              v-for="classes in clazzOptions"-->
      <!--              :key="classes.clazz"-->
      <!--              :label="classes.label"-->
      <!--              :value="classes.clazz">-->
      <!--            </el-option>-->
      <!--          </el-select>-->
      <!--        </el-col>-->
      <!--        <el-col :span="6">-->
      <!--          <el-input class='search-input' v-model="keyword" @change="search()" placeholder="请输入搜索内容"/>-->
      <!--        </el-col>-->
      <!--        <el-col :span="4" :offset="1">-->
      <!--          <el-button type="primary" icon="el-icon-search" @click="search()">搜索</el-button>-->
      <!--        </el-col>-->
      <!--      </el-row>-->
      <el-divider></el-divider>
      <el-table
        :data="tableData"
        v-loading="loading"
        stripe>
        <el-table-column
          prop="name"
          align="center"
          label="姓名">
        </el-table-column>
        <el-table-column
          prop="gender"
          align="center"
          label="性别">
        </el-table-column>
        <el-table-column
          prop="phone_number"
          align="center"
          label="联系电话">
        </el-table-column>
        <el-table-column
          prop="wx"
          align="center"
          label="微信号">
        </el-table-column>
        <el-table-column
          align="center"
          label="操作">
          <template  slot-scope="scope">
            <el-button
              type="primary"
              size="small"
              icon="el-icon-more"
              @click="showAllInfo(scope.row)"
              circle>
            </el-button>
          </template>
        </el-table-column>
      </el-table>
    </el-main>
    <el-footer>
      <el-pagination
        class="pagination"
        @current-change="handleCurrentChange"
        :current-page="currentAlumniPage"
        layout="prev, pager, next"
        :page-count="totalPage">
      </el-pagination>
    </el-footer>
  </el-container>
</template>

<script>
export default {
  name: 'ShowActivityParticipantsPage',
  data () {
    return {
      currentAlumniPage: 1,
      currentSemesterPage: 1,
      currentClassPage: 1,
      totalPage: 1,
      page_size: 20,
      class_state: true,
      loading: true,
      keyword: null,
      clazz: null,
      semester: null,
      tableData: [],
      semesterOptions: [],
      clazzOptions: [],
      currentActivity: []
    }
  },
  created () {
    this.currentActivity = this.$store.getters.getCurrentActivity
    console.log(this.currentActivity)
    if (this.axios) {
      this.findAllAlumni()
      // this.findAllSemester()
    }
  },
  methods: {
    goBack: function () {
      this.$router.go(-1)
    },
    showAllInfo (alumniInfo) {
      this.$store.dispatch('changeInfoOfAnAlumni', alumniInfo)
      this.$router.push('/show-all-information')
    },
    handleCurrentChange (currentAlumniPage) {
      this.currentAlumniPage = currentAlumniPage
      this.findAllAlumni()
    },
    findAllAlumni () {
      let params = {
        page_size: this.page_size,
        page: this.currentAlumniPage,
        activity_id: this.currentActivity.id
      }
      this.axios.get('/activity/student', { params }).then((response) => {
        if (response.data.code === '2000') {
          this.tableData = response.data.data.results
          for (let i = 0; i < this.tableData.length; i++) {
            if (this.tableData[i].gender === 0) {
              this.tableData[i].gender = '男'
            } else {
              this.tableData[i].gender = '女'
            }
          }
          this.loading = false
          this.$forceUpdate()
        } else {
          this.$message({
            type: 'error',
            message: '网络繁忙，请稍候重试',
            duration: 2000
          })
        }
      }).catch((error) => {
        console.log(error)
        this.$message({
          type: 'error',
          message: '网络繁忙，请稍候重试',
          duration: 2000
        })
      })
    },
    // findAllSemester: function () {
    //   let params = {
    //     page_size: this.page_size,
    //     page: this.currentSemesterPage
    //   }
    //   this.axios.get('/semester/semester', { params }).then((response) => {
    //     if (response.data.code === '2000') {
    //       let results = response.data.data.results
    //       this.semesterOptions[0] = {
    //         semester: null,
    //         label: '全部学期',
    //         id: 0
    //       }
    //       for (let i = 1; i < results.length + 1; i++) {
    //         this.semesterOptions[i] = {
    //           semester: results[i - 1]['period_semester'],
    //           label: results[i - 1]['subject'],
    //           id: results[i - 1]['id']
    //         }
    //       }
    //       this.$forceUpdate()
    //     } else {
    //       this.$message({
    //         type: 'error',
    //         message: '网络繁忙，请稍候重试',
    //         duration: 2000
    //       })
    //     }
    //   }).then((response) => {
    //     console.log(error)
    //     this.$message({
    //       type: 'error',
    //       message: '网络繁忙，请稍候重试',
    //       duration: 2000
    //     })
    //   })
    // },
    // findAllClass: function () {
    //   this.clazzOptions = []
    //   this.clazz = null
    //   if (this.semester === null) {
    //     this.class_state = true
    //   } else {
    //     this.class_state = false
    //   }
    //   let params = {
    //     page_size: this.page_size,
    //     page: this.currentClassPage,
    //     semester_id: this.semester
    //   }
    //   this.axios.get('/clazz/clazz', { params }).then((response) => {
    //     if (response.data.code === '2000') {
    //       let results = response.data.data.results
    //       for (let i = 0; i < results.length; i++) {
    //         this.clazzOptions[i] = {
    //           clazz: results[i]['id'],
    //           label: results[i]['name']
    //         }
    //       }
    //       this.search()
    //       this.$forceUpdate()
    //     } else {
    //       this.$message({
    //         type: 'error',
    //         message: '网络繁忙，请稍候重试',
    //         duration: 2000
    //       })
    //     }
    //   }).then((response) => {
    //     console.log(error)
    //     this.$message({
    //       type: 'error',
    //       message: '网络繁忙，请稍候重试',
    //       duration: 2000
    //     })
    //   })
    // },
    search () {
      let params = {
        page_size: this.page_size,
        page: this.currentAlumniPage,
        name: this.keyword,
        semester_id: this.semester,
        clazz_id: this.clazz
      }
      this.axios.get('/student/student', { params }).then((response) => {
        this.tableData = response.data.data.results
        for (let i = 0; i < this.tableData.length; i++) {
          if (this.tableData[i].gender === 0) {
            this.tableData[i].gender = '男'
          } else {
            this.tableData[i].gender = '女'
          }
        }
        this.loading = false
        this.$forceUpdate()
      })
    }
  }
}
</script>

<style scoped>
  .search-input {
    width: 340px;
    margin-right: 30px;
    margin-left: 130px;
  }

  .pagination {
    text-align: center;
  }
</style>
