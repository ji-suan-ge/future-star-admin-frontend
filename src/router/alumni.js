import ShowAlumniInformationPage from '../page/alumni-management/ShowAlumniInformationPage'
import EditAlumniInformationPage from '../page/alumni-management/EditAlumniInformationPage'
import ShowAlumniEditHistoryPage from '../page/alumni-management/ShowAlumniEditHistoryPage'
import ShowAlumniHistoryInformationPage from '../page/alumni-management/ShowAlumniHistoryInformationPage'

export default [
  {
    path: '/show-all-information',
    name: 'ShowAlumniInformationPage',
    component: ShowAlumniInformationPage
  },
  {
    path: '/edit-alumni-information',
    name: 'EditAlumniInformationPage',
    component: EditAlumniInformationPage
  },
  {
    path: '/show-edit-history',
    name: 'ShowAlumniEditHistoryPage',
    component: ShowAlumniEditHistoryPage
  },
  {
    path: '/show-history-information',
    name: 'ShowAlumniHistoryInformationPage',
    component: ShowAlumniHistoryInformationPage
  }
]
