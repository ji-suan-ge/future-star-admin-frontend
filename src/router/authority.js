import AddAdministratorPage from '../page/authority-management/AddAdministratorPage'

export default [
  {
    path: '/add-administrator',
    name: 'AddAdministratorPage',
    component: AddAdministratorPage
  }
]
