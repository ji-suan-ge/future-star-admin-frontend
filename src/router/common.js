import LoginPage from '../page/LoginPage'
import HomePage from '../page/HomePage'
import homeChildren from './home-children'

export default [
  {
    path: '/',
    name: 'Root',
    redirect: '/login'
  },
  {
    path: '/login',
    name: 'LoginPage',
    component: LoginPage
  },
  {
    path: '/home',
    name: 'HomePage',
    component: HomePage,
    redirect: '/home/welcome',
    children: [
      ...homeChildren
    ]
  }
]
