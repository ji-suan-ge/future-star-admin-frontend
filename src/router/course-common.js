import DetailOfSemesterPage from '../page/course-common/DetailOfSemesterPage'
import DetailOfClassPage from '../page/course-common/DetailOfClassPage'
import DetailOfCoursePage from '../page/course-common/DetailOfCoursePage'
import DetailOfTeacherPage from '../page/course-common/DetailOfTeacherPage'
import DetailOfSubCoursePage from '../page/course-common/DetailOfSubCoursePage'
import EditSemesterInfoPage from '../page/course-common/EditSemesterInfoPage'
import EditClassInfoPage from '../page/course-common/EditClassInfoPage'
import EditCourseInfoPage from '../page/course-common/EditCourseInfoPage'
import EditTeacherInfoPage from '../page/course-common/EditTeacherInfoPage'
import AddCoursePage from '../page/course-common/AddCoursePage'
import AddCourseResourcePage from '../page/course-common/AddCourseResourcePage'
import StudentInClass from '../page/course-common/StudentInClass'

export default [
  {
    path: '/semester-detail',
    name: 'DetailOfSemesterPage',
    component: DetailOfSemesterPage
  },
  {
    path: '/class-detail',
    name: 'DetailOfClassPage',
    component: DetailOfClassPage
  },
  {
    path: '/course-detail',
    name: 'DetailOfCoursePage',
    component: DetailOfCoursePage
  },
  {
    path: '/teacher-detail',
    name: 'DetailOfTeacherPage',
    component: DetailOfTeacherPage
  },
  {
    path: '/sub-course-detail',
    name: 'DetailOfSubCoursePage',
    component: DetailOfSubCoursePage
  },
  {
    path: '/edit-semester-info',
    name: 'EditSemesterInfoPage',
    component: EditSemesterInfoPage
  },
  {
    path: '/edit-class-info',
    name: 'EditClassInfoPage',
    component: EditClassInfoPage
  },
  {
    path: '/edit-course-info',
    name: 'EditCourseInfoPage',
    component: EditCourseInfoPage
  },
  {
    path: '/edit-teacher-info',
    name: 'EditTeacherInfoPage',
    component: EditTeacherInfoPage
  },
  {
    path: '/add-course',
    name: 'AddCoursePage',
    component: AddCoursePage
  },
  {
    path: '/add-course-resource',
    name: 'AddCourseResourcePage',
    component: AddCourseResourcePage
  },
  {
    path: '/student-in-class',
    name: 'StudentInClass',
    component: StudentInClass
  }
]
