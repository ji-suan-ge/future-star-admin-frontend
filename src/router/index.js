import Vue from 'vue'
import VueRouter from 'vue-router'
import LoginPage from '../page/LoginPage'
import HomePage from '../page/HomePage'
import EnrollmentManagementPage from '../page/EnrollmentManagementPage'
import CourseManagementPage from '../page/CourseManagementPage'
import ActivityManagementPage from '../page/ActivityManagementPage'
import AlumniManagementPage from '../page/AlumniManagementPage'
import AuthorityManagementPage from '../page/AuthorityManagementPage'
import CreateActivityPage from '../page/CreateActivityPage'
import AddAdministrator from '../page/AddAdministrator'
import WelcomePage from '../page/WelcomPage'

Vue.use(VueRouter)

const routes = [
  {
    path: '/',
    name: 'Root',
    redirect: '/login'
  },
  {
    path: '/login',
    name: 'LoginPage',
    component: LoginPage
  },
  {
    path: '/home',
    name: 'HomePage',
    component: HomePage,
    redirect: '/home/welcome',
    children: [
      {
        path: 'welcome',
        name: 'WelcomePage',
        component: WelcomePage
      },
      {
        path: 'enrollment-management',
        name: 'EnrollmentManagementPage',
        component: EnrollmentManagementPage
      },
      {
        path: 'course-management',
        name: 'CourseManagementPage',
        component: CourseManagementPage
      },
      {
        path: 'activity-management',
        name: 'ActivityManagementPage',
        component: ActivityManagementPage
      },
      {
        path: 'alumni-management',
        name: 'AlumniManagementPage',
        component: AlumniManagementPage
      },
      {
        path: 'authority-management',
        name: 'AuthorityManagementPage',
        component: AuthorityManagementPage
      },
      {
        path: 'add-administrator',
        name: 'AddAdministrator',
        component: AddAdministrator
      }
    ]
  },
  {
    path: '/create-activity',
    name: 'CreateActivityPage',
    component: CreateActivityPage
  }
]

const router = new VueRouter({
  routes
})

export default router
