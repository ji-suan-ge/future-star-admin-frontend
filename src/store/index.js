import Vue from 'vue'
import Vuex from 'vuex'

Vue.use(Vuex)

export default new Vuex.Store({
  state: {
    activeIndexOfNavigation: '0',
    activeNameOfCourse: 'first',
    activeNameOfEnrollment: 'first',
    activeNameOfActivity: 'first',
    activeSemesterOfCourse: '',
    semesters: '',
    currentSemester: '',
    classes: '',
    currentClass: '',
    courses: '',
    currentCourse: '',
    infoOfAnAlumni: null
  },
  mutations: {
    changeActiveIndexOfNavigation (state, activeIndex) {
      state.activeIndexOfNavigation = activeIndex
    },
    changeActiveNameOfEnrollment (state, activeName) {
      state.activeNameOfEnrollment = activeName
    },
    changeActiveNameOfCourse (state, activeName) {
      state.activeNameOfCourse = activeName
    },
    changeActiveNameOfActivity (state, activeName) {
      state.activeNameOfActivity = activeName
    },
    changeActiveSemesterOfCourse (state, activeID) {
      state.activeSemesterOfCourse = activeID
    },
    changeSemesters (state, semesters) {
      state.semesters = semesters
    },
    changeInfoOfAnAlumni (state, alumniInfo) {
      state.infoOfAnAlumni = alumniInfo
    },
    changeCurrentSemester (state, semester) {
      state.currentSemester = semester
    },
    changeClasses (state, classes) {
      state.classes = classes
    },
    changeCurrentClass (state, currentClass) {
      state.currentClass = currentClass
    },
    changeCourses (state, courses) {
      state.courses = courses
    },
    changeCurrentCourse (state, currentCourse) {
      state.currentCourse = currentCourse
    },
    initializeStateWhileLogin (state) {
      state.activeIndexOfNavigation = '0'
      state.activeNameOfCourse = 'first'
      state.activeNameOfEnrollment = 'first'
      state.activeNameOfActivity = 'first'
      state.activeSemesterOfCourse = ''
      state.semesters = ''
      state.currentSemester = ''
      state.classes = ''
      state.currentClass = ''
      state.courses = ''
      state.currentCourse = ''
      state.infoOfAnAlumni = null
    },
    initializeStateWhileSwitch (state) {
      state.activeNameOfCourse = 'first'
      state.activeNameOfEnrollment = 'first'
      state.activeNameOfActivity = 'first'
      state.activeSemesterOfCourse = ''
      state.semesters = ''
      state.currentSemester = ''
      state.classes = ''
      state.currentClass = ''
      state.courses = ''
      state.currentCourse = ''
      state.infoOfAnAlumni = null
    }
  },
  actions: {
    changeActiveIndexOfNavigation (context, activeIndex) {
      context.commit('changeActiveIndexOfNavigation', activeIndex)
    },
    changeActiveNameOfEnrollment (context, activeName) {
      context.commit('changeActiveNameOfEnrollment', activeName)
    },
    changeActiveNameOfCourse (context, activeName) {
      context.commit('changeActiveNameOfCourse', activeName)
    },
    changeActiveNameOfActivity (context, activeName) {
      context.commit('changeActiveNameOfActivity', activeName)
    },
    changeActiveSemesterOfCourse (context, activeID) {
      context.commit('changeActiveSemesterOfCourse', activeID)
    },
    changeSemesters (context, semesters) {
      context.commit('changeSemesters', semesters)
    },
    changeInfoOfAnAlumni (context, alumniInfo) {
      context.commit('changeInfoOfAnAlumni', alumniInfo)
    },
    changeCurrentSemester (context, semester) {
      context.commit('changeCurrentSemester', semester)
    },
    changeClasses (context, classes) {
      context.commit('changeClasses', classes)
    },
    changeCurrentClass (context, currentClass) {
      context.commit('changeCurrentClass', currentClass)
    },
    changeCourses (context, courses) {
      context.commit('changeCourses', courses)
    },
    changeCurrentCourse (context, currentCourse) {
      context.commit('changeCurrentCourse', currentCourse)
    },
    initializeStateWhileLogin (context) {
      context.commit('initializeStateWhileLogin')
    },
    initializeStateWhileSwitch (context) {
      context.commit('initializeStateWhileSwitch')
    }
  },
  getters: {
    getActiveIndexOfNavigation (state) {
      return state.activeIndexOfNavigation
    },
    getActiveNameOfEnrollment (state) {
      return state.activeNameOfEnrollment
    },
    getActiveNameOfCourse (state) {
      return state.activeNameOfCourse
    },
    getActiveNameOfActivity (state) {
      return state.activeNameOfActivity
    },
    getActiveSemesterOfCourse (state) {
      return state.activeSemesterOfCourse
    },
    getSemesters (state) {
      return state.semesters
    },
    getInfoOfAnAlumni (state) {
      return state.infoOfAnAlumni
    },
    getCurrentSemester (state) {
      return state.currentSemester
    },
    getClasses (state) {
      return state.classes
    },
    getCurrentClass (state) {
      return state.currentClass
    },
    getCourses (state) {
      return state.courses
    },
    getCurrentCourse (state) {
      return state.currentCourse
    }
  },
  modules: {
  }
})
