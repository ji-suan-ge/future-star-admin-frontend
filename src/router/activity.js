import CreateActivityPage from '../page/CreateActivityPage'
import ShowActivityParticipantsPage from '../page/ShowActivityParticipantsPage'
import EditExistingActivityPage from '../page/EditExistingActivityPage'
import ShowActivityInformationPage from '../page/ShowActivityInformationPage'

export default [
  {
    path: '/create-activity',
    name: 'CreateActivityPage',
    component: CreateActivityPage
  },
  {
    path: '/show-participants',
    name: 'ShowActivityParticipantsPage',
    component: ShowActivityParticipantsPage
  },
  {
    path: '/edit-activity',
    name: 'EditExistingActivityPage',
    component: EditExistingActivityPage
  },
  {
    path: '/show-activity-details',
    name: 'ShowActivityInformationPage',
    component: ShowActivityInformationPage
  }
]
