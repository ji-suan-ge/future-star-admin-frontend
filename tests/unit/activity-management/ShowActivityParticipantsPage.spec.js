import { createLocalVue, mount } from '@vue/test-utils'
import ShowActivityParticipantsPage from '../../../src/page/activity-management/ShowActivityParticipantsPage'
import store from '../../../src/store'
import router from '../../../src/router'
import Vuex from 'vuex'
import ElementUI from 'element-ui'
import VueRouter from 'vue-router'

describe('ShowActivityParticipantsPage', () => {
  const localVue = createLocalVue()
  localVue.use(Vuex)
  localVue.use(ElementUI)
  localVue.use(VueRouter)
  const wrapper = mount(ShowActivityParticipantsPage, {
    store,
    router,
    localVue
  })
  test('快照测试', async () => {
    await expect(wrapper.html()).toMatchSnapshot()
  })
  test('是一个 Vue 实例', async () => {
    await expect(wrapper.isVueInstance()).toBeTruthy()
  })
})
