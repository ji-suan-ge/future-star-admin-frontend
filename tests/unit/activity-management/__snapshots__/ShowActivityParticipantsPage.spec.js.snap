// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowActivityParticipantsPage 快照测试 1`] = `
<section class="el-container is-vertical">
  <main class="el-main">
    <div class="el-page-header">
      <div class="el-page-header__left"><i class="el-icon-back"></i>
        <div class="el-page-header__title">返回</div>
      </div>
      <div class="el-page-header__content">已参与校友名单</div>
    </div>
    <div class="el-divider el-divider--horizontal">
      <!---->
    </div>
    <div class="search-input el-input">
      <!----><input type="text" autocomplete="off" placeholder="请输入搜索内容" class="el-input__inner">
      <!---->
      <!---->
      <!---->
      <!---->
    </div> <button type="button" class="el-button el-button--primary">
      <!----><i class="el-icon-search"></i><span>搜索</span></button>
    <div class="el-divider el-divider--horizontal">
      <!---->
    </div>
    <div class="el-table el-table--fit el-table--striped el-table--scrollable-x el-table--enable-row-hover el-table--enable-row-transition">
      <div class="hidden-columns">
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div><button type="button" class="el-button el-button--primary el-button--small is-circle">
            <!----><i class="el-icon-more"></i>
            <!----></button></div>
      </div>
      <div class="el-table__header-wrapper">
        <table cellspacing="0" cellpadding="0" border="0" class="el-table__header" style="width: 400px;">
          <colgroup>
            <col name="el-table_1_column_1" width="80">
            <col name="el-table_1_column_2" width="80">
            <col name="el-table_1_column_3" width="80">
            <col name="el-table_1_column_4" width="80">
            <col name="el-table_1_column_5" width="80">
          </colgroup>
          <thead class="">
            <tr class="">
              <th colspan="1" rowspan="1" class="el-table_1_column_1  is-center   is-leaf">
                <div class="cell">姓名</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_2  is-center   is-leaf">
                <div class="cell">性别</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_3  is-center   is-leaf">
                <div class="cell">联系电话</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_4  is-center   is-leaf">
                <div class="cell">微信号</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_5  is-center   is-leaf">
                <div class="cell">操作</div>
              </th>
            </tr>
          </thead>
        </table>
      </div>
      <div class="el-table__body-wrapper is-scrolling-left">
        <table cellspacing="0" cellpadding="0" border="0" class="el-table__body" style="width: 400px;">
          <colgroup>
            <col name="el-table_1_column_1" width="80">
            <col name="el-table_1_column_2" width="80">
            <col name="el-table_1_column_3" width="80">
            <col name="el-table_1_column_4" width="80">
            <col name="el-table_1_column_5" width="80">
          </colgroup>
          <tbody>
            <tr class="el-table__row">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">13456327654</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">11111111</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row el-table__row--striped">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">23456327654</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">22222222</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">33456327654</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">33333333</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row el-table__row--striped">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">43456327654</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">44444444</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">53456327654</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">55555555</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button></div>
              </td>
            </tr>
            <!---->
          </tbody>
        </table>
        <!---->
        <!---->
      </div>
      <!---->
      <!---->
      <!---->
      <!---->
      <div class="el-table__column-resize-proxy" style="display: none;"></div>
    </div>
  </main>
  <footer class="el-footer" style="height: 60px;">
    <div class="pagination el-pagination"><span class="el-pagination__total">共 5 条</span><span class="el-pagination__sizes"><div class="el-select el-select--mini"><!----><div class="el-input el-input--mini el-input--suffix"><!----><input type="text" readonly="readonly" autocomplete="off" placeholder="请选择" class="el-input__inner"><!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><i class="el-select__caret el-input__icon el-icon-arrow-up"></i><!----><!----><!----><!----><!----></span>
      <!----></span>
      <!---->
      <!---->
    </div>
    <div class="el-select-dropdown el-popper" style="display: none;">
      <div class="el-scrollbar" style="">
        <div class="el-select-dropdown__wrap el-scrollbar__wrap el-scrollbar__wrap--hidden-default">
          <ul class="el-scrollbar__view el-select-dropdown__list">
            <!---->
            <li class="el-select-dropdown__item selected"><span>10条/页</span></li>
            <li class="el-select-dropdown__item"><span>20条/页</span></li>
            <li class="el-select-dropdown__item"><span>30条/页</span></li>
            <li class="el-select-dropdown__item"><span>40条/页</span></li>
          </ul>
        </div>
        <div class="el-scrollbar__bar is-horizontal">
          <div class="el-scrollbar__thumb" style="transform: translateX(0%); webkit-transform: translateX(0%);"></div>
        </div>
        <div class="el-scrollbar__bar is-vertical">
          <div class="el-scrollbar__thumb" style="transform: translateY(0%); webkit-transform: translateY(0%);"></div>
        </div>
      </div>
      <!---->
    </div>
    </div></span><button type="button" disabled="disabled" class="btn-prev"><i class="el-icon el-icon-arrow-left"></i></button>
    <ul class="el-pager">
      <li class="number active">1</li>
      <!---->
      <!---->
      <!---->
    </ul><button type="button" disabled="disabled" class="btn-next"><i class="el-icon el-icon-arrow-right"></i></button><span class="el-pagination__jump">前往<div class="el-input el-pagination__editor is-in-pagination"><!----><input type="number" autocomplete="off" min="1" max="1" class="el-input__inner"><!----><!----><!----><!----></div>页</span></div>
  </footer>
</section>
`;
